api/tests/core/users/test_api.py

```diff
@@ -1323,6 +1323,20 @@ def test_can_save_login_device(self):
         assert login_device.os == "iOS"
         assert login_device.location == None
 
+    def test_should_return_login_history(self):
+        user = users_factories.UserFactory(email="py@test.com")
```

Comment:
les autres commits ont améliorés les tests mais celui ci n'a pas profité des améliorations : `test` + `should` et `email`

Reply:
⸮ quoi ?! tu es humain ? tu fais des erreurs ?!

pas la honte du tout !